pie/storage/__init__.py

```diff
@@ -0,0 +1,142 @@
+from __future__ import annotations
+
+from pydoc import locate
+from typing import Any
+
+from nextcord.ext import commands
+
+from pie.storage.database import StorageData
+
+
+def get(module: commands.Cog, guild_id: int, key: str, default_value=None) -> Any:
+    """Get data from persistant DataStorage base on module and guild.
+    For saving global data (non-guild related) set guild_id to 0.
+
+    If value is not found in DB or it's type is unknown,
+    it returns default_value, which if not set is None.
```

Comment:
> which if not set is None

tat does not make grammatical sense. Did you mean

> `None` by default.

?

Reply:
Yep, that's what I meant.